plugins/json-logger/webpack.config.js

```diff
@@ -0,0 +1,44 @@
+const path = require("path");
+const webpack = require("webpack");
+const CopyWebpackPlugin = require("copy-webpack-plugin");
+const nodeExternals = require("webpack-node-externals");
```

Comment:
Remove this line as we need all plugin to bundle all their dependencies
more info: https://github.com/amplication/amplication/issues/7150